MAINTAINERS.md

```diff
@@ -0,0 +1,13 @@
+| Name | GitHub | RocketChat |
+| --- | --- | --- |
+| Adam Ludvik | aludvik | adamludvik |
+| Andi Gunderson | agunde406 | agunde |
+| Anne Chenette | chenette | achenette |
+| Boyd Johnson | boydjohnson | boydjohnson |
+| Dan Middleton | dcmiddle | Dan |
+| Darian Plumb | dplumb94 | dplumb |
+| James Mitchell | jsmitchell | jsmitchell |
+| Kelly Olson | kelly\_ | ineffectualproperty |
```

Comment:
Looks like Github and RocketChat got switched. Github is ineffectualproperty, RocketChat is kelly_

Reply:
Fixed.